fe/src/project/AspNetCore/areas/Products/components/ProductDetail/ProductDetail.js

```diff
@@ -215,7 +215,7 @@ function ProductDetail(props) {
                     labels={props.sidebar}
                 />
             </div>
-            <CarouselGrid items={props.productVariants}/>
+            <CarouselGrid items={props.productVariants} className="pt-6"/>
```

Comment:
**suggestion:** there is a padding-top: 0; in carousel grid css.  Maybe it would be enough to delete this line? Would it break something?

Reply:
This will revert to our earlier change to the spacing between sections on the home page